### PR TITLE
Add entity schema rest resource

### PR DESCRIPTION
**Pull request by mattgrill**
## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/435

## Screenshot / UI changes

- Create a new resource for entity schema at `/admin-api/entity-schema/{entityTypeID}/{nid}`

## Testing instructions

- Ensure that `/admin-api/entity-schema/{entityTypeID}/{nid}` returns the correct schema.

## Todo

- How should set permissions, currently you have to give permission to the resource explicitly
- Tests

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -74,7 +74,7 @@ jobs:
                 composer create-project jsdrupal/drupal-admin-ui-demo -s dev --prefer-dist
                 cd drupal-admin-ui-demo
                 mv ~/project drupal-admin-ui
-                composer config repositories.repo-name path "./drupal-admin-ui/admin_ui_support"
+                composer config repositories.repo-name path "drupal-admin-ui/admin_ui_support"
```
Comment (dawehner) [inline .circleci/config.yml]: @webflo Do you know how we can correctly link in a folder of the current project?

Comment (webflo) [reply .circleci/config.yml]: Have you tried to not run ``mv ~/project drupal-admin-ui`` and use ``composer config repositories.repo-name path "~/project/admin_ui_support" instead?

Comment (webflo) [reply .circleci/config.yml]: ``../project/admin_ui_support`` should work as well according to the docs on https://getcomposer.org/doc/05-repositories.md#path

**Comment by webflo** [comment]
🎉

**Pull request merged**